samples/trigger-customizations/event-display/main.go

```diff
@@ -42,6 +42,7 @@ func display(event cloudevents.Event) {
 }
 
 func main() {
+	flag.Parse()
```

Comment:
this change is unrelated to this PR. Just something I saw was missing while testing out changes.